Code/Python/circular_queue.py

```diff
@@ -0,0 +1,134 @@
+"""
+Circular Queue
+Problem Statement: Circular Queue is a linear data structure which follows FIFO principle
+                   The difference is that the last position is connected back to the first position to make a circle.
+                   Perform insertion, deletion and traversal operations on Circular Queue.
+"""
+
+
+class CircularQueue:
+  
+    # Defining and initializing the class
+    def __init__(self, quesize):
+        # initializing start and end of queue with -1
+        self.quehead = -1
+        # initializing queue with zero value
+        self.queue = [0 for k in range(quesize)]
+        self.quetail = -1
+        # size of queue as quesize
+        self.size = quesize
+
+
+    def traverse(self):
+        # checking if the queue is Underflow
+        if(self.quehead == -1):
+            print ("Underflow")
+
+        #else printing the elements of queue
+        else:
+            print("Elements in Queue: ")
+            for k in range(self.quehead, self.quetail + 1):
+                print(self.queue[k], end = " ")
+                
+

```

Comment:
Decrease double line space to single line spaces

Reply:
@Rohan27s isn't there a rule to add 2 line spaces between functions.. for differentiating them in python.